README.md

```diff
@@ -1,15 +1,47 @@
-#Repository for Innovation Labs infographic website
-To run using npm http-server
+# Infographic Infrastructure Automation Demo
+
+## Prerequisistes
+* NodeJS
+* npm
+
+## Running
 Run: 
-```js
-npm install -g http-server
-http-server website
+```bash
+npm install
+npm start
 ```
 
-This will start up a local http server using the `website` directory as the root context of the server.
+This will start up a local http server to host the fron using the `website` directory as the root context of the front end application. This local devleopment server will also host the API. In OpenShift-based environments, the front end and back end are hosted as separate applications, using the templates in the `templates` directory.
+
+## Important paths
+
+- `/index.html`: The public-facing infographic page. Clicking the "build" button takes the user to the Open Innovation Labs website.
+- `/internal.html`: The "internal" version of infographic. Clicking the "build" button kicks off an Ansible job.
+- `/stack`: The node.js API that the front end hits when the user clicks the "build" button. This API passes the request on to Ansible.
```

Comment:
Same comment as above -- if kept, I'd like to change `... request on to Ansible.` to `... request on to Ansible Tower (per configuration params below)`.